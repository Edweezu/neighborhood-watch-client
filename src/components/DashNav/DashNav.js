import React from 'react'
import { NavLink } from 'react-router-dom'
import MainContext from '../../contexts/MainContext'
import Footer from '../Footer/Footer'

class DashNav extends React.Component {


    static contextType = MainContext

    render () {

        // const { categories } = this.context

     return (
            <section className='DashNav'>
                {/* <div className='menu-container'>
                    <div className='main-nav-link'>
                        {categories.map(category => (
                            <NavLink className='DashNav__folderLink' to={`/category/${category.id}`}>
                                {category.name}
                            </NavLink>
                        ))}
                    </div>
                </div> */}
                 <div className='menu-container'>
                    <div className='main-nav-link'>
                        <NavLink
                            className='DashNav__folderLink'
                            to={`/category/1`}
                        >
                            All Posts
                        </NavLink>
                    </div>
                </div>
                <div className='menu-container'>
                    <div className='main-nav-link'> 
                        <NavLink
                            className='DashNav__folderLink'
                            to={`/category/2`}
                        >
                            Crime and Alerts
                        </NavLink>
                    </div>
                </div>
                <div className='menu-container'>
                    <div className='main-nav-link'>
                        <NavLink
                            className='DashNav__folderLink'
                            to={`/category/3`}
                        >
                            Upcoming Events
                        </NavLink>
                    </div>
                </div>
                <div className='menu-container'>
                    <div className='main-nav-link'>
                        <NavLink
                            className='DashNav__folderLink'
                            to={`/category/4`}
                        >
                            Lost and Found
                        </NavLink>
                    </div>
                </div>
                <div className='Nav__logout'>
                    <div className='main-nav-link '>
                        <NavLink
                            className='DashNav__folderLink'
                            to={`/`}
                        >
                            Logout
                        </NavLink>
                    </div>
                </div> 
                <Footer />   
            </section>
                      
        )
    }
}

export default DashNav