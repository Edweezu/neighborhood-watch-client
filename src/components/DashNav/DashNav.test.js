import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import DashNav from './DashNav'


describe('DashNav component', () => {
    it('renders the given component', () => {
        const wrapper = shallow(<DashNav />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})