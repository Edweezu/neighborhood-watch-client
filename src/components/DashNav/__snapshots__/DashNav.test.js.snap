// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashNav component renders the given component 1`] = `
<section
  className="DashNav"
>
  <div
    className="menu-container"
  >
    <div
      className="main-nav-link"
    >
      <NavLink
        className="DashNav__folderLink"
        to="/category/1"
      >
        All Posts
      </NavLink>
    </div>
  </div>
  <div
    className="menu-container"
  >
    <div
      className="main-nav-link"
    >
      <NavLink
        className="DashNav__folderLink"
        to="/category/Crime and Alerts"
      >
        Crime and Alerts
      </NavLink>
    </div>
  </div>
  <div
    className="menu-container"
  >
    <div
      className="main-nav-link"
    >
      <NavLink
        className="DashNav__folderLink"
        to="/category/Upcoming Events"
      >
        Upcoming Events
      </NavLink>
    </div>
  </div>
  <div
    className="menu-container"
  >
    <div
      className="main-nav-link"
    >
      <NavLink
        className="DashNav__folderLink"
        to="/category/Lost and Found"
      >
        Lost and Found
      </NavLink>
    </div>
  </div>
  <div
    className="Nav__logout"
  >
    <div
      className="main-nav-link "
    >
      <NavLink
        className="DashNav__folderLink"
        to="/"
      >
        Logout
      </NavLink>
    </div>
  </div>
  <Footer />
</section>
`;
